gradle/libs.versions.toml

```diff
@@ -1,5 +1,5 @@
 [versions]
-agp = "8.0.1"
+agp = "7.4.2"
```

Comment:
Please undo this. I understand that the project is not working in IDEA but that's a Google/Jetbrains problem which they should solve.

Once AGP hits stable, I'll update the template to match the latest stable as there is an Android Studio version out that can open it.

Reply:
Do you mind if I revert this and replace it with a note in the README about this known issue?

Reply:
Yes please do. Also this is a recurrent issue of IntelliJ so not sure what we can do here actually

Reply:
I've documented this in README.md